components/gitpod-cli/cmd/credential-helper.go

```diff
@@ -28,6 +33,15 @@ var credentialHelper = &cobra.Command{
 	Hidden: true,
 	Run: func(cmd *cobra.Command, args []string) {
 		action := args[0]
+		log.SetOutput(io.Discard)
+		f, err := os.OpenFile(os.TempDir()+"/gitpod-git-credential-helper.log", os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644)
+		if err == nil {
+			defer f.Close()
+			log.SetOutput(f)
+		}
+		log.Infof("gp credential.helper %s", action)
+		log.Infof("isTheiaIDE() + %t", isTheiaIDE())
```

Comment:
nit: in general, please prefer structured logging over `logf`, i.e. `log.WithField("action", action)`
For the credential helper that's much less relevant because we're not producing JSON logs/it's not a service. But the use of `logf` stands out nonetheless.

Reply:
Deleted these lines entirely.